shop/src/main/java/ro/msg/learning/shop/model/Order.java

```diff
@@ -0,0 +1,42 @@
+package ro.msg.learning.shop.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.List;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Table(name = "order_product")
+@EqualsAndHashCode(callSuper = true)
+public class Order extends MainEntity<Integer> {
+    @ManyToOne
+    @JoinColumn(name = "shipped_from")
+    private Location location;
+    @ManyToOne
+    @JoinColumn(name = "customer")
+    private Customer customer;
+    private LocalDate createdAt;
+    private String country;
+    private String city;
+    private String county;
+    private String streetAddress;
+
+    @OneToMany(mappedBy = "order", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
The default fetch type is lazy so you can remove that settings

Reply:
I deleted them as they were unnecessary and might've caused some unexpected issues